### DISCUSSION TITLE
How do I store strings in the clipboard?

### DISCUSSION THREAD

**Discussion by Seikaijyu** [AvaloniaUI/Avalonia#8355] [Q&A] [answered]
I want to save the string to the clipboard, but I can't find the relevant API

**Comment by MarkusKgit** [comment]
Check out [TextCopy](https://github.com/CopyText/TextCopy)

**Comment by jp2masa** [comment, marked as answer]
https://docs.avaloniaui.net/docs/input/clipboard

**Reply by MarkusKgit** [reply]
Doh, how did I miss this 😆

**Reply by Seikaijyu** [reply]
I didn't notice the API. Thank you very much

**Comment by miguel-rodriguez-cimino** [comment]
Was this API moved? I don't see a Clipboard property under Application.Current

**Reply by jp2masa** [reply]
https://docs.avaloniaui.net/docs/next/concepts/toplevel#clipboard
